apm-agent-plugins/apm-jdbc-plugin/src/test/java/co/elastic/apm/agent/jdbc/AbstractJdbcInstrumentationTest.java

```diff
@@ -408,6 +408,7 @@ private Span assertSpanRecorded(String rawSql, boolean preparedStatement, long e
         Db db = span.getContext().getDb();
         assertThat(db.getStatement()).isEqualTo(rawSql);
         DatabaseMetaData metaData = connection.getMetaData();
+        assertThat(db.getInstance()).isEqualToIgnoringCase(connection.getCatalog());
```

Comment:
[question] why should we need to ignore case here ? Given the instrumentation also calls `getCatalog` the values must be identical.

Reply:
The surrounding asserts also used isEqualToIgnoringCase, although it is not necessary (e.g. `assertThat(db.getType()).isEqualToIgnoringCase("sql")`). So I did the same.